### PR TITLE
[13.0][MIG]hr_contract_multi_job

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module was written to extend the functionality of employees contracts to support multiple jobs of the employee assigned on the same contract.

An example is a construction worker who makes different types of jobs for the same company like bricklaying, electricity, carpentry.

Also in restaurants, a waiter can also work as a barman.

### CODE REVIEW

File: hr_contract_multi_job/readme/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To install this module, you need to:
+
+* clone the branch 12.0 of the repository https://github.com/OCA/hr
```
Comment (jarroyomorales) [inline hr_contract_multi_job/readme/INSTALL.rst]: Change to 13.0.

**Comment by ps-tubtim** [comment]
@jarroyomorales Thank you for review.

File: hr_contract_multi_job/README.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
+======================
```
Comment (alexey-pelykh) [inline hr_contract_multi_job/README.rst:1]: Either keep this file up-to-date using generator, or don't include it.

File: hr_contract_multi_job/readme/INSTALL.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+To install this module, you need to:
```
Comment (alexey-pelykh) [inline hr_contract_multi_job/readme/INSTALL.rst]: This install instruction is common for all modules, there's no value including in everywhere.

File: hr_contract_multi_job/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+On every employee contract you can set multiple jobs, and mark one of the job
+as the main job (the main job will be written afterwards in the job_id field).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (alexey-pelykh) [inline hr_contract_multi_job/readme/USAGE.rst]: Link to runbot is added by default, this should be removed

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Comment by ps-tubtim** [comment]
@alexey-pelykh Thanks for suggestions.

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Comment by ps-tubtim** [comment]
@Saran440 Could you review this?

**Review by Saran440 — APPROVED** [review]
Functional Test. LGTM :+1: 
FYI String `Main Job Position` in table job position different from odoo12 a little bit.

**Comment by Saran440** [comment]
@ps-tubtim Don't squash legit translation commits, authored by their translators. Could you please fix it?

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration

**Pull request merged**